app/src/interfaces/map/map.vue

```diff
@@ -108,7 +108,13 @@ import { useAppStore } from '@/stores/app';
 import { TranslateResult, useI18n } from 'vue-i18n';
 
 import { getBasemapSources, getStyleFromBasemapSource } from '@/utils/geometry/basemap';
-
+maplibre.setRTLTextPlugin(
+	'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js',
```

Comment:
What's the usage license on this plugin? I know that mapbox-gl-js itself @mapbox/mapbox-gl-geocoder, which prevents us from using it in the platform

Reply:
Honestly, I don't know, but we can replace api.mapbox.com URL with the URL in the maplibre docs which is supposed to be licensed under the [3-Clause BSD license](https://github.com/maplibre/maplibre-gl-js/blob/main/LICENSE.txt) ?

Reply:
Is there a way to load the code from the locally included copy of maplibre? That way we don't have to do a third-party request (which helps in covering GDPR), and it ensures that we know where the code originated, so we can make sure the license is OK 🙂

Reply:
Unfortunately, it's not included in any of mapbox-gl/maplibre-gl or any already installed package on directus, we are using the URL from this project https://www.npmjs.com/package/@mapbox/mapbox-gl-rtl-text and the license on GitHub is here [License](https://github.com/mapbox/mapbox-gl-rtl-text/blob/main/LICENSE.md)

Reply:
Granted I'm also not sure how I feel about loading via a 3rd party request, but I assume such approach would also require either another entry in the current default Content Security Policy config, or explicit docs for it (or maybe ideally this plugin is only loaded when it is set to RTL via an interface option) 🤔

That being said, it might also not be as straightforward to install it as an additional package due to https://github.com/mapbox/mapbox-gl-rtl-text/issues/27 since we're currently using mapbox-gl `^2.9.1`, unless maybe resorting to something like [pnpm overrides](https://pnpm.io/package_json#pnpmoverrides).

Reply:
@azrikahar Ay that's an unfortunate problem. Is the last released version in that repo the same as the one on the live link? There might be a reason they're requiring <2

Reply:
> Is the last released version in that repo the same as the one on the live link?

Yeah seems like it is.

> There might be a reason they're requiring <2

Judging from the observations below:

- A user force installed `2.5.1` in their setup and have yet to face any problem: https://github.com/mapbox/mapbox-gl-rtl-text/issues/27#issue-1034511191

- A mapbox maintainer suggested to drop the peerDependencies completely and mentioned they do not intend to break compatibility even for v3: https://github.com/mapbox/mapbox-gl-rtl-text/pull/28#pullrequestreview-789492376

- The official migration guide stated v2 is backwards compatible (with a few exceptions, but I assume they are unrelated based on their descriptions, along with the 1st point above tested it for `2.5.1`): https://docs.mapbox.com/mapbox-gl-js/guides/migrate-to-v2/

perhaps it was a safety net just in case v2 _does_ contain breaking change, since the library is last updated/released _before_ v2 was released 🤔 However this is 100% still solely based on the observations above, and may not actually be correct.

Reply:
Hi @rijkvanzanten and @azrikahar ,
What is the status of this PR? Is there any update from my side that should be done?
Thanks!